internal/cmd/commands/scopescmd/rotate_keys.go

```diff
@@ -0,0 +1,118 @@
+package scopescmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/boundary/api"
+	"github.com/hashicorp/boundary/api/scopes"
+	"github.com/hashicorp/boundary/internal/cmd/base"
+	"github.com/mitchellh/cli"
+	"github.com/mitchellh/go-wordwrap"
+	"github.com/posener/complete"
+)
+
+var (
+	_ cli.Command             = (*RotateKeysCommand)(nil)
+	_ cli.CommandAutocomplete = (*RotateKeysCommand)(nil)
+)
+
+type RotateKeysCommand struct {
+	*base.Command
+}
+
+func (c *RotateKeysCommand) Synopsis() string {
+	return wordwrap.WrapString("Rotate the keys within a scope", base.TermWidth)
+}
+
+func (c *RotateKeysCommand) Help() string {
+	return base.WrapForHelpText([]string{
+		"Usage: boundary scopes rotate-keys [args]",
+		"",
+		"  Generate new keys for a scope and all nested scopes. Example:",
```

Comment:
Is it true that it generates new keys for nested scopes too? I'm not sure?

Reply:
I thought it was but I don't think it is anymore. the new comment is mostly taken from the description given by [go-kms-wrapping](https://github.com/hashicorp/go-kms-wrapping/blob/main/extras/kms/kms.go#L436-L437), so should be accurate